pyproject.toml

```diff
@@ -34,4 +34,8 @@ capitalisation_policy = "upper"
 [tool.pydocstyle]
 ignore = ["D100", "D101", "D102", "D107", "D104", "D213", "D407", "D413"]
 
-[tool.black]
\ No newline at end of file
+[tool.black]
+
+[[tool.mypy.overrides]]
+module = "geopandas.*" # no stubs available
+ignore_missing_imports = true
```

Comment:
to confirm, we don't want `ignore_missing_imports` to be a global mypy setting?

I'm not familiar with configuring mypy but I think at the moment, this setting is module-specific to `geopandas` and it's submodules

Reply:
@damonmcc Correct, we only want this to affect geopandas, which has no type stubs available (at least not in our current version). It's currently the only dcpy lib where that's the case.